queries/c/injections.scm

```diff
@@ -8,6 +8,10 @@
   (#match? @injection.content "/\*!([a-zA-Z]+:)?re2c")
   (#set! injection.language "re2c"))
 
+((comment) @injection.content
+  (#match? @injection.content "/\*(!|<|\*)[^\w]")
```

Comment:
Shouldn't all of these patterns be `^pattern`

Reply:
/** /*! and /*< signify a potential doxygen blob

I added a not word after to avoid double injecting w/ re2c